docker-compose.yml

```
@@ -0,0 +1,62 @@
+version: '2'
+services:
+  web:
+    image: fpfis/httpd-php-dev:7.3
+    working_dir: /var/www/html
+    ports:
+      - 8080:8080
+    volumes:
+      - .:/var/www/html # Non Mac users.
+      # - nfsmount:/var/www/html # Mac Users with the nfsmount volume.
+    environment:
+      # XDEBUG_CONFIG: "client_host=172.17.0.1" # Non-Mac users.
+      # XDEBUG_CONFIG: remote_host=host.docker.internal remote_port=9000 remote_enable=1, idekey=VSCODE
+      XDEBUG_CONFIG: "client_host=172.17.0.1 client_port=9000  remote_host=172.17.0.1 remote_port=9000 remote_enable=1, idekey=VSCODE"
+      # XDEBUG_CONFIG: "client_host=host.docker.internal" # Mac users.
+      XDEBUG_MODE: "debug"
+      PHP_IDE_CONFIG: "serverName=Docker"
+      # Enable step debugging for all PHP request. See ./README.md#step-debugging for more information.
+      XDEBUG_SESSION: "VSCODE"
```

```suggestion
      # XDEBUG_SESSION: 1
```
again, add your own config in override